components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/test/java/org/wso2/carbon/identity/application/authentication/framework/AbstractFrameworkTest.java

```diff
@@ -43,6 +44,7 @@ protected AuthenticationContext getAuthenticationContext(ServiceProvider service
         authenticationContext.setTenantDomain("test_domain");
         authenticationContext.setCurrentStep(1);
         authenticationContext.setContextIdentifier(UUID.randomUUID().toString());
+        authenticationContext.setRequestType(FrameworkConstants.RequestType.CLAIM_TYPE_OIDC);
```

Comment:
remove protocol specific logic 

add a new test to check the not configured claims configuration